AppCenterDistribute/AppCenterDistributeTests/Util/MSMockReachabilityUtil.m

```diff
@@ -0,0 +1,39 @@
+#import "MSMockReachabilityUtil.h"
+#import "MSTestFrameworks.h"
+
+static NSString *kMSNetworkReachabilityChangedNotificationName = @"kMSNetworkReachabilityChangedNotification";
+
+@interface MSMockReachabilityUtil ()
+
+@property(nonatomic) id mockReachabilityUtil;
+
+@end
+
+@implementation MSMockReachabilityUtil
+
+@synthesize currentNetworkStatus = _currentNetworkStatus;
+
+- (instancetype)init {
+  self = [super init];
+  if (self) {
+    
+    // Mock MS_Reachability shared method to return this instance.
+    _mockReachabilityUtil = OCMClassMock([MS_Reachability class]);
```

Comment:
You mock the class after call `init` of it. It looks wrong.

Reply:
Isolate mocking code from init, is it right?

Reply:
Changing the super class solves this issue too.